discord/guild.py

```diff
@@ -1911,6 +1911,36 @@ async def create_custom_emoji(
         data = await self._state.http.create_custom_emoji(self.id, name, img, roles=roles, reason=reason)
         return self._state.store_emoji(self, data)
 
+    async def delete_custom_emoji(
+        self,
+        emoji: abc.Snowflake,
+        *,
+        reason: Optional[str] = None
+    ) -> None:
```

Comment:
```suggestion
    async def delete_emoji(self, emoji: abc.Snowflake, *, reason: Optional[str] = None) -> None:
```

This isn't long enough to have in many lines. Also the name is inconsistent with the public API.